aws/static_site/main.tf

```diff
@@ -0,0 +1,74 @@
+resource "aws_s3_bucket" "mod" {
+  bucket = "${var.bucket_name}"
+  acl    = "public-read"
+
+  policy = <<EOF
```

Comment:
Surprised you need a policy

Reply:
I dont recall why that needs to be there. I think I got it from a blog post a long time ago.